docs/index.md

```diff
@@ -61,7 +61,7 @@ To use the **Tanzu Mission Control provider** for Tanzu Mission Control Self-Man
 [prepapre-cluster-for-tmc-sm]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/1.0/tanzumc-sm-install/prepare-cluster.html
 
 -> **Note:**
-Tanzu Mission Control Provider supports Tanzu Mission Control Self-Managed from **v1.1.9** onwards.
+Tanzu Mission Control Provider supports Tanzu Mission Control Self-Managed from **v1.2.0** onwards.
```

Comment:
I believe we support SM v1.0 (current & first public release) onwards. So , IMO, stating SM version support isn't needed - as terraform providers support starts from the first public SM release onwards.

Reply:
I have added the TMC SM version as suggested. @vmw-vjn in my opinion the reason being when the next version of TMC SM is available it will be explicit that terraform provider support is on v1.0 even if there are any new changes introduced in later versions of TMC SM which is not supported by terraform until we release that support.